README.md

```diff
@@ -58,9 +58,9 @@ OpenMRS Dictionary Manager|
 [![Organization Management](https://github.com/openmrs/openmrs-ocl-client/actions/workflows/organisation-management.yml/badge.svg)](https://github.com/openmrs/openmrs-ocl-client/actions/workflows/organisation-management.yml)
 Loading into EMR ____ Coming soon
 
-OCL Module (Subscription Module)
+OCL Module (Subscription Module) |
 -----|
-[OCL Module](https://ci.openmrs.org/browse/OCLM-OCLM) ![Build Status](https://ci.openmrs.org/plugins/servlet/wittified/build-status/OCLM-OCLM)
+[![OCL Subscription Module](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/ocl-subscription-module.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/ocl-subscription-module.yml)
```

Comment:
How ocl-subscription instead of Ocl-subscription-module

Reply:
@sherrif10 I don't understand the above.

Reply:
i meant how would it be when its **ocl-subscription**  instead of **ocl-subscription-module**

Reply:
Why remove the word module 🤔 ?

Reply:
This is fine too, we can mantain it there probably this makes it more readable.Thanks @hadijahkyampeire